packages/triutils/src/Trilinos_Util_ReadHpc2Epetra.cpp

```diff
@@ -207,20 +213,22 @@ void Trilinos_Util_ReadHpc2Epetra_internal(
           std::cout << "Process "<< rank <<" of "
                     << size <<" getting RHS " << i
                     << std::endl;
-#ifndef NDEBUG
-        cnt =
-#endif
+//#ifndef NDEBUG
+         auto cnt =
+//#endif
         fscanf(in_file, "%lf %lf %lf",&xt, &bt, &xxt);
+	 (void) cnt;
         assert(cnt > 0);
         int cur_local_row = map->LID(i);
         (*x)[cur_local_row] = xt;
         (*b)[cur_local_row] = bt;
         (*xexact)[cur_local_row] = xxt;
       } else {
-#ifndef NDEBUG
-        cnt =
-#endif
+//#ifndef NDEBUG
+         auto cnt =
+//#endif
         fscanf(in_file, "%lf %lf %lf",vp, vp, vp); // or thrown away
```

Comment:
You should just be able to write `(void) fscanf(...);`.

Reply:
Oh, this should not have been in this pull request... Fixed now.